### PR TITLE
Do-not-show-console

### LINKED ISSUE #57: Do not show QueryConsole when in grid or vis mode
Query Console doesn't show anythong useful, it only occupies half of the screen. Error should be shown using vscode notifications

### PR DESCRIPTION
**Pull request by Yanpas**
Fixes #57

### CODE REVIEW

File: src/client/client.ts [hunk 1]
```diff
@@ -87,7 +87,7 @@ export function activate(context: ExtensionContext): void {
     window.registerTreeDataProvider('q-explorer', qRoot);
     window.registerTreeDataProvider('q-history', qHistory);
     qHistory.refresh();
-    QueryConsole.createOrShow();
+    QueryConsole.get();
```
Comment (jshinonome) [inline src/client/client.ts:90]: Please add a show after this line. I want vscode to show q output when vscode-q is activated.

**Comment by Yanpas** [comment]
Fixed

**Comment by jshinonome** [comment]
I like some of your changes, but you didn't remove those parts that I pointed out. I made a change myself, so no need anymore.

**Comment by Yanpas** [comment]
I only saw one comment/note about showing console on start, maybe I've missed something

**Comment by Yanpas** [comment]
You are still showing console to the user every time you call "createOrGet", so nothing changed really

**Comment by jshinonome** [comment]
Sorry, you are right. I forgot to remove a line. Give a try 2.4.9

**Comment by jshinonome** [comment]
fixed it, v2.5.0

**Pull request closed without merging**